content/en/docs/contributing/style-guide.md

```diff
@@ -47,11 +47,29 @@ are properly written and use the original capitalization. For example, write
 [`.textlintrc.yml`](https://github.com/open-telemetry/opentelemetry.io/blob/main/.textlintrc.yml)
 file.
 
-## Markdown standards
+## Markdown
+
+Site pages are written in the Markdown syntax supported by the [Goldmark]
+Markdown renderer. For the full list of supported Markdown extensions, see
+[Goldmark].
+
+You can also use the following extensions:
+
+- [GitHub-flavored Markdown][GFM] (GFM) [alerts][gfm-alerts]
+- [Emojis]. For the complete list of available emojis, see [Emojis] from the
+  Hugo docs.
+
+[Emojis]: https://gohugo.io/quick-reference/emojis/
```

Comment:
I'm not sure we want to use emojis though (outside of a few functional ones like checkboxes, etc.)...